modules/files/module.yml

```
@@ -1,12 +1,14 @@
 name: files
-shortdesc: Copy files to your image at build time
+shortdesc: Remove and copy files to your image at build time
```

```suggestion
shortdesc: Copy files to your image or remove unwanted ones from it at build time
```

You place "remove" first in every modified documentation sentence. Copying files is still the primary purpose of the module, so that should be kept as the first thing mentioned.